case_studies/README.md

```diff
@@ -12,21 +30,22 @@ Here is what the directory structure looks like:
 ```
 .
 ├── club_1
-|   ├── event_a
+|   ├── event_1
 |   |   ├── planning.md
 |   |   └── post-mortem.md
-|   ├── event_b
+|   ├── event_1
 |   |   ├── planning.md
 |   |   └── post-mortem.md
 ├── club_2
 |...
 ```
 
-Each club's directory contains sub-directories for each of the club's events (i.e
-club fair, first club meeting, etc.)
+Each club's directory contains sub-directories for each of the club's events
+(i.e club fair, first club meeting, etc.)
```

Comment:
I don't think this is proper use of "i.e.", see http://theoatmeal.com/comics/ie.